phoneApp/js/main.js

```diff
@@ -0,0 +1,119 @@
+const users = [
+    {
+        name: 'Иван',
+        lastName: 'Петров',
+        email: 'ivanpetrov@gmail.com'
+    },
+    {
+        name: 'Сергей',
+        lastName:'Сергеев',
+        email: 'sergeysergeev@gmail.com'
+    },
+    {
+        name: 'Иван',
+        lastName: 'Петров',
+        email: 'ivanpetrov@gmail.com'
+    },
+    {
+        name: 'Александр',
+        lastName: 'Александров',
+        email: 'alex@gmail.com'
+    },
+    {
+        name: 'Алекс',
+        lastName: 'Смирнов',
+        email: 'smirnov@gmail.com'
+    },
+    {
+        name: 'Сергей',
+        lastName: 'Волков',
+        email: 'srgvolk@gmail.com'
+    },
+    {
+        name: 'Мария',
+        lastName: 'Шарапова',
+        email: 'mari@gmail.com'
+    },
+    {
+        name: 'Александр',
+        lastName: 'Винник',
+        email: 'smirnov@gmail.com'
+    },
+];
+const colums = ['Name', 'LastName', 'Email'];
+const app  = {
+    users,
+    colums,
+    render() {
+        document.body.innerHTML += this.renderHeader() +this.renderMain() + this.renderFooter();
```

Comment:
const contactPageLayot = this.renderHeader() +this.renderMain() + this.renderFooter();

`document.body.innerHTML` not sure I like to use document.body.

You have to create an additional tag at index.html and call it, for example,
```html
<div id="mountNode"></div>
```
And update content only inside such node. updating the whole document.body is too risky